buildscripts/cmake/SetupBuildEnvironment.cmake

```diff
@@ -34,7 +34,6 @@ elseif(CC_IS_MSVC)
     set(CMAKE_C_FLAGS_DEBUG             "/MT /Zi /Ob0 /Od /RTC1")
     set(CMAKE_C_FLAGS_RELEASE           "/MT /O2 /Ob2")
     set(CMAKE_C_FLAGS_RELWITHDEBINFO    "/MT /Zi /O2 /Ob1")
-    set(CMAKE_EXE_LINKER_FLAGS          "/DYNAMICBASE:NO")
```

Comment:
How about
```
if (MUSE_MODULE_AUDIO_JACK)
    set(CMAKE_EXE_LINKER_FLAGS          "/DYNAMICBASE:NO")
endif (MUSE_MODULE_AUDIO_JACK)
```
to be on the save side?

Reply:
I would prefer not doing that for now, and only do that when it turns out to be really necessary in practice. Things may have changed since when that crash was actual; and disabling ASLR is not a very nice thing to do, for the mentioned security reasons.

Reply:
The corresponding issue against Jack are still open